bundles/org.openhab.binding.enphase/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,228 @@
+		</channels>
+
+		<properties>
+			<property name="partNumber"/>
+		</properties>
+
+		<representation-property>serialnumber</representation-property>
+
+		<config-description>
+			<parameter name="serialnumber" type="text" pattern="[0-9]{12}" required="true">
+				<label>Serial Number</label>
+				<description>The serial number of the inverter</description>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<thing-type id="relay">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="envoy"/>
+		</supported-bridge-type-refs>
+
+		<label>Relay Controller</label>
+		<description>Network system relay controller</description>
+
+		<channels>
+			<channel id="relay" typeId="relay"/>
+			<channel id="line1Connected" typeId="line-connected">
+				<label>Line 1 Connection Status</label>
+			</channel>
+			<channel id="line2Connected" typeId="line-connected">
+				<label>Line 2 Connection Status</label>
+			</channel>
+			<channel id="line3Connected" typeId="line-connected">
+				<label>Line 3 Connection Status</label>
+			</channel>
+			<channel id="status" typeId="status"/>
+			<channel id="producing" typeId="producing"/>
+			<channel id="communicating" typeId="communicating"/>
+			<channel id="provisioned" typeId="provisioned"/>
+			<channel id="operating" typeId="operating"/>
+		</channels>
+
+		<properties>
+			<property name="partNumber"/>
+		</properties>
+
+		<representation-property>serialnumber</representation-property>
+
+		<config-description>
+			<parameter name="serialnumber" type="text" pattern="[0-9]{12}" required="true">
+				<label>Serial Number</label>
+				<description>The serial number of the inverter</description>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<!-- Envoy gateway channels -->
+	<channel-group-type id="envoy-data">
+		<label>Envoy Data</label>
+		<channels>
+			<channel id="wattHoursToday" typeId="watt-hours-today"/>
+			<channel id="wattHoursSevenDays" typeId="watt-hours-seven-days"/>
+			<channel id="wattHoursLifetime" typeId="watt-hours-lifetime"/>
+			<channel id="wattsNow" typeId="watts-now"/>
+		</channels>
+	</channel-group-type>
+
+	<channel-type id="watt-hours-today">
+		<item-type>Number:Energy</item-type>
+		<label>Watt Hours Today</label>
+		<description>Watt hours today</description>
+		<state pattern="%d %unit%" readOnly="true"/>
+	</channel-type>
+	<channel-type id="watt-hours-seven-days">
+		<item-type>Number:Energy</item-type>
+		<label>Watt Hours 7 Days</label>
+		<description>Watt Hours of the last 7 days</description>
+		<state pattern="%d %unit%" readOnly="true"/>
+	</channel-type>
+	<channel-type id="watt-hours-lifetime">
+		<item-type>Number:Energy</item-type>
+		<label>Watt Hours Lifetime</label>
+		<description>Watt Hours Lifetime</description>
+		<state pattern="%d %unit%" readOnly="true"/>
+	</channel-type>
+	<channel-type id="watts-now">
+		<item-type>Number:Power</item-type>
+		<label>Watt Now</label>
+		<description>Watt Now</description>
```

Comment:
Suggest changing to Watts Now to be consistent with the channel name. The labels are used when 'creating equipment from thing'.

Reply:
Yes should be changed. I've changed it a bit to be similar to what enphase itself names it.

Reply:
What is your thoughts on having a channel called `wattsNow` that will auto create an item with `Foo_LatestPower` as the name? Will this confuse more people then just me?

Reply:
The naming is a ui specific feature I think. If another translation would be made for a channel label it will, I assume, already give another name. So I don't think it should be an issue the name is different. Which does trigger me that I should update the readme. So if people whould look into the readme they see the same text as in the xml file.